src/main.rs

```diff
@@ -406,6 +400,15 @@ fn main() {
         }
     };
 
+    // Show config file and path, pass through
+    if args.flag_config_path {
+        eprintln!("Warning: The --config-path flag is deprecated, use --show-paths instead");
+        show_config_path();
+    }
+    if args.flag_show_paths {
+        show_paths(&config.directories.custom_pages_dir);
+    }
+
```

Comment:
I don't know if I asked this before, but what is the rationale behind this change?

Reply:
Because now show_paths needs a reference to the custom_pages_dir from the config. So had to move this logic to after `Config::load`